src/imagery/i.eodag/i.eodag.py

```diff
@@ -37,66 +35,83 @@
 # % answer: S1_SAR_GRD
 # % guisection: Filter
 # %end
-
+# %option G_OPT_V_MAP
+# % description: If not given then current computational extent is used
+# % label: Name of input vector map to define Area of Interest (AOI)
+# % required: no
+# % guisection: Region
+# %end
+# %option
```

Comment:
```suggestion

# %option
```

Reply:
Not sure if the style guide says something about spacing among options, but I do feel it is easy to read when there's an empty line between options